openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -425,6 +416,23 @@ const TeamDetailsV1 = ({
     updateTeamHandler(updatedTeamData);
   };
 
+  const fetchPermissions = async () => {
+    getEntityPermission(ResourceEntity.TEAM, currentTeam.id)
```

Comment:
Can we use the async/await pattern here?

Reply:
made suggested changes